### ISSUE TITLE
after checking out ebpf-for-windows repository clang doesn't build with includes

### ISSUE THREAD

**Issue by jsturtevant** [microsoft/ebpf-for-windows#1409]
### Describe the bug

I am not 100% sure if this is really a bug or if I am missing a step but if I have a program with `#include "bpf_helpers.h"` and I compile it with includes from this project `clang -I .\ebpf-for-windows\include\`  I get

```
#include "libbpf/src/bpf_helpers.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

This looks related to the change in https://github.com/microsoft/ebpf-for-windows/pull/999

### OS information

PS C:\Users\User> cmd /c ver

Microsoft Windows [Version 10.0.22000.978]

### Steps taken to reproduce bug

create file:

```
#include "bpf_helpers.h"
        
int func()
{
    return 0;
}
```

Clone this repo and run:

```
clang -I .\ebpf-for-windows\include\ -target bpf -Werror -O2 -c .\bpf.c -o bpf.o
#include "libbpf/src/bpf_helpers.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

### Expected behavior

build to work from dev branch

### Actual outcome

error

### Additional details

If I pull the nuget package ( `nuget install eBPF-for-Windows`) and reference it the build passes:

```
clang -I .\eBPF-for-Windows.0.4.0\build\native\include\ -target bpf -Werror -O2 -c .\bpf.c -o bpf.o
```

**Comment by jsturtevant** [comment]
The `libbpf/src/bpf_helpers.h` is missing from https://github.com/microsoft/ebpf-for-windows/tree/main/include which is why it is failing.  

It seems like this script maps it into the nuget folder but I don't see it explicitly: 
https://github.com/microsoft/ebpf-for-windows/blob/main/scripts/create_package_data.bat

Maybe need to include another folder? It seems these includes are being brought in by external package "bpftool": https://github.com/microsoft/ebpf-for-windows/blob/52817e82961c16d0abe976685b8b553f1d923fef/scripts/create_package_data.bat#L80-L83

**Comment by dthaler** [comment]
Fixed in PR #1414
